lib/cards/post_card.dart

```diff
@@ -14,9 +14,69 @@ class PostCard extends StatelessWidget {
     required this.item,
   });
 
+  String getDate() {
+    DateTime now = DateTime.now();
+    DateTime date = item.timeCreated.toDate();
+    if (now.difference(date).inDays == 0) {
+      return "Today";
+    }
+    if (now.difference(date).inDays == 1) {
+      return "Yesterday";
+    }
+    return "${date.month}-${date.day}";
+  }
+
   @override
   Widget build(BuildContext context) {
     final theme = Theme.of(context).extension<AppTheme>()!;
+    final BoxDecoration bodyDecoration = BoxDecoration(
+      color: theme.medium,
```

Comment:
White text is difficult to read on the green background, could we try black text?